### ISSUE TITLE
[Adaptive Cards-Designer]: Accessible name should be defined in English for Arabic button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8885]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

Test environment:  

OS: Win11  

OS Version: 24H2 (OS Build 26100.1)    

URL:[Adaptive Cards](https://adaptivecards.io/) 

Edge dev Browser version: 125.0.2506.0 (Official build) dev (64-bit)  

Screen reader: Narrator. 

 

Repro Steps: 

1. Open URL in the Edge browser & sign in with valid credentials. 
2. It will open the Adaptive cards home page 

3. Press tab key and navigate to the 'New Card' button and press enter key to activate it.

4. Navigate to 'Arabic' language under Please select your language and observe the issue.

**Actual Result:**  

The Aria label of the button is defined in different languages instead of English language and when focus lands on that control, screen reader is announcing as bullet list button collapsed 

**Expected Result:** 

The Aria label of the button should be defined in English language so that screen reader announces the proper information. 

**User Experience:**  

The screen reader users will face difficulties if the screen reader does not pronounce the correct language for the buttons.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/a2bfcabd-38bc-4a7f-989d-7e5a872cbbe6

![Code-Arabic-Aria label](https://github.com/microsoft/AdaptiveCards/assets/95913748/283b66e7-908e-4a61-9f69-502f21164bab)
![Accessible name should be defined in English for Arabic button](https://github.com/microsoft/AdaptiveCards/assets/95913748/c9ab6bb1-7e28-43c5-a4f5-af4974fdda77)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

![Image](https://github.com/user-attachments/assets/20933cd2-edfe-410e-981f-353b40dc5817)
